components/server/src/websocket/websocket-connection-manager.ts

```diff
@@ -373,6 +375,19 @@ class GitpodJsonRpcProxyFactory<T extends object> extends JsonRpcProxyFactory<T>
     }
 
     protected async onRequest(method: string, ...args: any[]): Promise<any> {
+        const userId = this.clientMetadata.userId;
+        return runWithContext(
+            {
+                requestId: v4(),
```

Comment:
(out of scope here) Would be great to discuss the structure of this shape a bit, and evolbe the old "OWI" approch.

Reply:
Yes, we should discuss our approach to tracing

Reply:
btw what does OWI stand for?